### PR TITLE
Translation of Methodology to Hindi

### PR DESCRIPTION
**Pull request by PraveenPal4232**
- Added hi/2019/methodology.html
- Added hi/2020/methodology.html
- Move chapter templates

For issue #1402

### CODE REVIEW

File: src/templates/hi/2019/methodology.html [hunk 1]
```diff
@@ -0,0 +1,428 @@
+{% extends "base/methodology.html" %}
+
+{% block title %}कार्यप्रणाली | HTTP Archive द्वारा Web Almanac{% endblock %}
+
+{% block description %}यह बताता है कि 2019 Web Almanac को कैसे एक साथ रखा गया था: किस तरह से डेटासेट्स और टूल्स का उपयोग किया गया और इस परियोजना को शुरु किया गया था।{% endblock %}
+
+{% block twitter_image_alt %}{{ year }} Web Almanac की कार्यप्रणाली{% endblock %}
+
+
```
Comment (bazzadp) [inline src/templates/hi/2019/methodology.html:9]: Remove extra line to keep line numbers consistent with English

```suggestion
```

File: src/templates/hi/2019/methodology.html [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+        </p>
+
+        <p>
+          Web Almanac में प्रस्तुत सभी मैट्रिक्स BigQuery पर डेटासेट का उपयोग कर सार्वजनिक रूप से प्रतिलिपि प्रस्तुत करने योग्य हैं। आप हमारी <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2019">GitHub रिपॉजिटरी</a> में सभी अध्यायों द्वारा उपयोग किए गए क्वेरी को देख सकते हैं।
+        </p>
+
+        <p class="note">
+          कृपया ध्यान दें कि इनमें से कुछ क्वेरी काफी बड़ी हैं और आप के प्रयोग के लिए <a href="https://cloud.google.com/bigquery/pricing">महंगे</a> हो सकते हैं, क्योंकि BigQuery को टेराबाइट द्वारा बिल किया जाता है। अपने खर्च को नियंत्रित करने में सहायता के लिए, टिम कडलेक की पोस्ट <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a> देखें।
```
Comment (bazzadp) [inline src/templates/hi/2019/methodology.html:79]: I get a weird message when I use the [HTML validator](https://validator.w3.org/nu/) on this line:

Nu validation message

Which apparently [is some encoding issue](https://stackoverflow.com/a/5466610/2144578).

I ran that text through a normaliser and got this and no it doesn't complain. Is this the same?

```suggestion
          कृपया ध्यान दें कि इनमें से कुछ क्वेरी काफी बड़ी हैं और आप के प्रयोग के लिए <a href="https://cloud.google.com/bigquery/pricing">महंगे</a> हो सकते हैं, क्योंकि BigQuery को टेराबाइट द्वारा बिल किया जाता है। अपने खर्च को नियंत्रित करने में सहायता के लिए, टिम कडलेक की पोस्ट <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a> देखें।
```

File: src/templates/hi/2020/methodology.html [hunk 3]
```diff
@@ -0,0 +1,476 @@
+{% extends "base/methodology.html" %}
+
+{% block title %}कार्यप्रणाली | HTTP Archive द्वारा Web Almanac{% endblock %}
+
+{% block description %}यह बताता है कि 2020 Web Almanac को कैसे एक साथ रखा गया था: किस तरह से डेटासेट्स और टूल्स का उपयोग किया गया और इस परियोजना को शुरु किया गया था।{% endblock %}
+
+{% block twitter_image_alt %}{{ year }} Web Almanac की कार्यप्रणाली{% endblock %}
+
+
```
Comment (bazzadp) [inline src/templates/hi/2020/methodology.html:9]:
```suggestion
```

File: src/templates/hi/2020/methodology.html [hunk 4]
```diff
@@ -0,0 +1,476 @@
...
+        </p>
+
+        <p>
+          Web Almanac में प्रस्तुत सभी मैट्रिक्स BigQuery पर डेटासेट का उपयोग कर सार्वजनिक रूप से प्रतिलिपि प्रस्तुत करने योग्य हैं। आप हमारी <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">GitHub रिपॉजिटरी</a> में सभी अध्यायों द्वारा उपयोग किए गए क्वेरी को देख सकते हैं।
+        </p>
+
+        <p class="note">
+          कृपया ध्यान दें कि इनमें से कुछ क्वेरी काफी बड़ी हैं और आप के प्रयोग के लिए <a href="https://cloud.google.com/bigquery/pricing">महंगे</a> हो सकते हैं, क्योंकि BigQuery को टेराबाइट द्वारा बिल किया जाता है। अपने खर्च को नियंत्रित करने में सहायता के लिए, टिम कडलेक की पोस्ट <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a> देखें।
```
Comment (bazzadp) [inline src/templates/hi/2020/methodology.html:84]:
```suggestion
          कृपया ध्यान दें कि इनमें से कुछ क्वेरी काफी बड़ी हैं और आप के प्रयोग के लिए <a href="https://cloud.google.com/bigquery/pricing">महंगे</a> हो सकते हैं, क्योंकि BigQuery को टेराबाइट द्वारा बिल किया जाता है। अपने खर्च को नियंत्रित करने में सहायता के लिए, टिम कडलेक की पोस्ट <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a> देखें।
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Looks good!

A couple of minor things to change and then we can merge this.

File: src/templates/hi/base_ebook.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{% extends "base/base_ebook.html" %}
+
+{% block title %}HTTP Archive द्वारा {{ year }}  Web Almanac{% endblock %}
+
```
Comment (bazzadp) [inline src/templates/hi/base_ebook.html:4]: I missed one:

```suggestion
```

File: src/templates/hi/2019/methodology.html (hunk 2 above)
Comment (PraveenPal4232) [reply src/templates/hi/2019/methodology.html]: Fixed

File: src/templates/hi/2020/methodology.html (hunk 4 above)
Comment (PraveenPal4232) [reply src/templates/hi/2020/methodology.html]: Fixed

File: src/templates/hi/base_ebook.html (hunk 5 above)
Comment (PraveenPal4232) [reply src/templates/hi/base_ebook.html:4]: Done

File: src/templates/hi/2020/methodology.html (hunk 3 above)
Comment (PraveenPal4232) [reply src/templates/hi/2020/methodology.html]: Done

File: src/templates/hi/2019/methodology.html (hunk 1 above)
Comment (PraveenPal4232) [reply src/templates/hi/2019/methodology.html]: Done

**Review by bazzadp — APPROVED** [review]
LGTM

Did you know you can just accept the suggested changes in GitHub? Though where there are a few it's sometimes easier to make the changes locally and do one commit and push.

Anyway all good now and merging.

**Comment by rviscomi** [comment]
> Though where there are a few it's sometimes easier to make the changes locally and do one commit and push.

I noticed that GitHub now supports batching suggested changes in the web UI too! Especially helpful when there are lots of comments :)

**Comment by rviscomi** [comment]
Thanks for working on this @PraveenPal4232 I'm excited to add support for a new language! 🚀

**Comment by PraveenPal4232** [comment]
> LGTM
> 
> Did you know you can just accept the suggested changes in GitHub? Though where there are a few it's sometimes easier to make the changes locally and do one commit and push.
 
 Yes, I know. But I prefer to do it manually. This way we have few and relevant commits for the same piece of code.

**Comment by PraveenPal4232** [comment]
> Thanks for working on this @PraveenPal4232 I'm excited to add support for a new language! 🚀

Welcome, @rviscomi. Pleasure is all mine :)
And Thanks to you to take this great insensitive for the web community.

**Pull request merged**